--- origsrc/tuxpaint-config-0.0.12/Makefile	2009-05-28 17:08:08.000000000 -0500
+++ src/tuxpaint-config-0.0.12/Makefile	2009-11-23 22:16:00.279233500 -0600
@@ -255,7 +255,7 @@ install-icon:
 install-bin:
 	@echo
 	@echo "...Installing program itself..."
-	@cp tuxpaint-config $(BIN_PREFIX)
+	@cp tuxpaint-config$(EXE_EXT) $(BIN_PREFIX)
 	@chmod a+rx,g-w,o-w $(BIN_PREFIX)/tuxpaint-config$(EXE_EXT)
 
 
